packages/vitest/src/node/reporters/json.ts

```diff
@@ -193,6 +193,9 @@ export class JsonReporter implements Reporter {
     if (!frame)
       return
 
-    return { line: frame.line, column: frame.column }
+    let file = ''
```

Comment:
I don't think having an empty string here is a good idea

Reply:
return file == '' case should be unreachable. Because there should be no error without stack
So it's mostly for convincing typescript that all is ok anyway
If there are line and column, but there is no file, then that's strange input to this function. Strange input - strange output

Reply:
If you just want to satisfy TS, you can use `!`